### ISSUE TITLE
Internal 500 error when prettying JSON response

### ISSUE THREAD

**Issue by onurhb** [ijpiantanida/talkback#17]
Given following:

```
var talkback = require("talkback");

var server = talkback({
  host: "https://jsonplaceholder.typicode.com",
  path: __dirname + "/tapes",
  port: 3002,
  debug: true
});

server.start();
```

I am getting response 500, when requesting `localhost:3002/todos/1` 
with header: `Content-Type: application/json`.

Removing [this method](https://github.com/ijpiantanida/talkback/blame/6a616493e1759656463374cf04089eeafe96b84a/src/tape.js#L66) fixes the problem.

I don't understand why this is necessary.

**Comment by ijpiantanida** [comment]
Can I ask you how you're making the request?

Normalizing is necessary to compare JSON tapes, since the original body formatting is lost when pretty printing.

**Comment by onurhb** [comment]
Just curl:

``` sh
$ curl --header "content-type:application/json" localhost:3002/todos/1 -I

HTTP/1.1 500 Internal Server Error
Date: Fri, 01 Feb 2019 17:19:33 GMT
Connection: keep-alive
```

**Comment by ijpiantanida** [comment]
Thanks for the info.
The error happened on HEAD requests.

I just published v1.8.1 with a fix.
